packages/strapi/package.json

```diff
@@ -1,26 +1,8 @@
 {
   "name": "strapi",
   "version": "3.0.0-beta.1",
-  "description": "An open source solution to create and manage your own API. It provides a powerful dashboard and features to make your life easier.",
+  "description": "An open source solution to create and manage your own API. It provides a powerful dashboard and features to make your life easier. Databases supported: MongoDB, MySQL, MariaDB, PostgreSQL, SQLite",
```

Comment:
"An open source headless CMS solution to create and manage your own API. It provides a powerful dashboard and features to make your life easier. Databases supported: MongoDB, MySQL, MariaDB, PostgreSQL, SQLite"

Shouldn't we mention something about being a headless CMS?

Reply:
Sounds good to me. I can try to get that in my PR.